src/notifier.js

```diff
@@ -160,7 +151,7 @@ NotifierPrototype._getLogArgs = function(args) {
     } else if (argT === 'date') {
       extraArgs.push(arg);
     } else if (argT === 'error' ||
-               arg.stack ||
+               arg instanceof Error ||
```

Comment:
Yup, I'm planning on a minor bump.

Relying on the stack property is not a great idea since it doesn't provide any guarantees that we will be able to get useful information from it. Also, if someone passes in a plain object with a stack property we were treating it as an error which is probably not what they would want. Also, IE9- doesn't provide a .stack property so we cannot rely on it.